tests/test_examples.py

```diff
@@ -709,11 +719,13 @@ def _create_command_line(
                 f"--learning_rate {lr}",
                 f"--per_device_train_batch_size {train_batch_size}",
                 f"--per_device_eval_batch_size {eval_batch_size}",
-                f" --num_train_epochs {num_epochs}",
+                f"--num_train_epochs {num_epochs}",
                 "--use_habana",
                 "--throughput_warmup_steps 3",
                 "--save_strategy no",
             ]
+            if not gaudi_config_name:
+                cmd_line.pop(cmd_line.index(f"--gaudi_config_name {gaudi_config_name}"))
```

Comment:
you mean if not None ?

Reply:
No, `gaudi_config_name` in this case for mixtral is an empty string "", please look at `test/utils.py`

Reply:
it still doesn't make sense, the expression only valuates to True if gaudi_config_name is "" ? 😅